python_modules/airline-demo/airline_demo/solids.py

```diff
@@ -40,25 +41,28 @@ def notebook_solid(name, inputs, outputs):
 
 
 # need a sql context w a sqlalchemy engine
-def sql_solid(name, select_statement, materialization_strategy, table_name=None):
+def sql_solid(name, select_statement, materialization_strategy, table_name=None, inputs=None):
     '''Return a new solid that executes and materializes a SQL select statement.
 
     Args:
         name (str): The name of the new solid.
         select_statement (str): The select statement to execute.
         materialization_strategy (str): Must be 'table', the only currently supported
             materialization strategy. If 'table', the kwarg `table_name` must also be passed.
-
     Kwargs:
         table_name (str): THe name of the new table to create, if the materialization strategy
-            is 'table'.
+            is 'table'. Default: None.
+        inputs (list[InputDefinition]): Inputs, if any, for the new solid. Default: None.
 
     Returns:
         function:
             The new SQL solid.
     '''
+    if inputs is None:
```

Comment:
you can use:
```
      inputs = check.opt_list_param(inputs, 'inputs', InputDefinition)
```

Reply:
yep yep, i thought about this and my only question was whether we want to encourage users to use `check`, will make the change